core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -116,7 +116,9 @@ object ApiVersion {
     // Introduce AllocateProducerIds (KIP-730)
     KAFKA_3_0_IV0,
     // Introduce ListOffsets V7 which supports listing offsets by max timestamp (KIP-734)
-    KAFKA_3_0_IV1
+    KAFKA_3_0_IV1,
+    // Introduced topic IDs to MetadataRequest
```

Comment:
Why are we bumping IBP? Metadata is not an inter-broker protocol, so I think bumping the metadata protocol suffices.

Reply:
You are right, we no longer need to check UNSUPPORTED_VERSION_ERROR, so it's unnecessary to bump IBP.